src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -358,11 +420,15 @@ private static async Task<int> Run(string url, string hostname, string dockerHos
 
                     var rcsb = new RelayConnectionStringBuilder(relayConnectionString);
 
+                    if (_certBasedAuth.HasValue())
+                    {
+                        options.TokenProvider = GetAadTokenProvider(_certTenantId.Value(), _certClientId.Value(), certificate);
```

Comment:
GetAadTokenProvider takes clientId, then tenantId as parameter order, so it is passed in incorrect order here

Reply:
I changed the definition of `GetAadTokenProvider` so that it takes the tenantId first. This matches how all of the other APIs are written.